MAVProxy/modules/lib/live_graph_ui.py

```diff
@@ -166,6 +166,13 @@ def on_redraw_timer(self, event):
             state.values = state.child_pipe.recv()
         if self.paused:
             return
+
+        if (type(state.values[0]) == list):
```

Comment:
Needs to work across all values, not just the first.

Reply:
I've moved the check into the for loop below